gantt-chart/test/jest.config.js

```diff
@@ -24,4 +24,5 @@ module.exports = {
     '\\.(css|less|sass|scss)$': '<rootDir>/test/mocks/styleMock.ts',
     '\\.(gif|ttf|eot|svg)$': '<rootDir>/test/mocks/fileMock.ts',
   },
+  testEnvironment: 'jsdom',
```

Comment:
Is this required?

Reply:
yes, jest version in dashboards changed, default env is node.js

Reply:
Should this be added to dashboards-reports as well?

Reply:
depends, reporting have jest for node.js env, need to check separately